docs/connector-development/connector-builder-ui/maintenance.md

```diff
@@ -0,0 +1,175 @@
+    def __init__(self):
+        super().__init__(**{"path_to_yaml": "manifest.yaml"})
+
+```
+
+Once this is done, commit your changes and push them to GitHub in a branch with a descriptive name.
+
+
+#### Through GitHub UI
+* Go to https://github.com/airbytehq/airbyte/branches, click on "New branch" and enter a descriptive name
+* Export the manifest from the Connector Builder, remove the `metadata` field, go to https://github.com/airbytehq/airbyte/tree/<your branch name>/airbyte-integrations/connectors/source-<name>/source_<name>/manifest.yaml, click on the pencil to "Edit this file", paste the Connector Builder manifest and commit this file
+* If `https://github.com/airbytehq/airbyte/tree/<your branch name>/airbyte-integrations/connectors/source-<name>/source_<name>/schemas` exists, click on the `...` on the top right side, select `Delete directory` and click on "Commit changes"
+* If `https://github.com/airbytehq/airbyte/tree/<your branch name>/airbyte-integrations/connectors/source-<name>/source_<name>/spec.yaml` exists, click on the `...` on the top right side, select `Delete file` and click on "Commit changes"
+* Given any file with .py except `source.py` extension under `https://github.com/airbytehq/airbyte/tree/<your branch name>/airbyte-integrations/connectors/source-<name>/source_<name>`, delete the file
+* For source.py, ensure that it looks like this:
+```
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+from airbyte_cdk.sources.declarative.yaml_declarative_source import YamlDeclarativeSource
+
+"""
+This file provides the necessary constructs to interpret a provided declarative YAML configuration file into
+source connector.
+
+WARNING: Do not modify this file.
+"""
+
+
+# Declarative Source
+class Source<name>(YamlDeclarativeSource):
+    def __init__(self):
+        super().__init__(**{"path_to_yaml": "manifest.yaml"})
+
+```
+
+## Validation
+
+Goal: Ensuring the changes work as expected and provide a safety net for the next modifications
+
+Expected output: CATs are green
+
+To ensure we can validate the changes on a connector, CATs are executed. A Connector Acceptance Test is a piece of software that emulates a worker (see architecture [here](../../understanding-airbyte/jobs.md)) and checks the output of commands that the connector is expected to implement. In order to configure these tests, we need to:  
+* Create tests scenarios in a sandbox environment: There is no streamlined process to do this. Each product has their own business rules that can't be captured in a unifying document. Given a new connector, this process might require to have pay for a subscription for the product in order to get credentials/workspace/etc... In order to be able to repopulate an API very easily, we suggest the use of a Postman collection or a script.
+* Configure what we want to be validated: Using `acceptance-test-config.yml`, you can toggle some of the validations that are performed. The most common cases are:
```

Comment:
What about linking to https://docs.airbyte.com/connector-development/testing-connectors/connector-acceptance-tests-reference/?